sgx/sync/src/condvar.rs

```diff
@@ -419,13 +419,16 @@ impl SgxCondvar {
 
     fn verify(&self, mutex: &SgxThreadMutex) {
         let addr = mutex as *const _ as usize;
-        match self.mutex.compare_and_swap(0, addr, Ordering::SeqCst) {
+        match self
+            .mutex
+            .compare_exchange(0, addr, Ordering::SeqCst, Ordering::SeqCst)
```

Comment:
This is due to `compare_and_swap` being deprecated. Please see https://doc.rust-lang.org/std/sync/atomic/struct.AtomicUsize.html#method.compare_exchange